src/pages/Admin/Details/ValidationSchema.ts

```diff
@@ -252,5 +252,5 @@ export const eventEditSchema = Yup.object().shape<EventInput>({
     .typeError("Minutes must be a number")
     .required("Minutes equired"),
   host: Yup.string().required("Host required"),
-  room: Yup.string().matches(/^(?!Select a room...$).*$/, "Room is required"),
+  room: Yup.string().required("Space is required"),
```

Comment:
[NOTE]
This might look OOS to the task of Dropdown. But OK if change to new dropddown makes the form not break.